flask_resty/testing.py

```diff
@@ -30,30 +30,37 @@ def get_meta(response):
 def assert_value(actual, expected):
     if isinstance(expected, Mapping):
         assert isinstance(actual, Mapping)
-        for k, v in expected.items():
-            assert_value(actual.get(k, None), v)
+        # Unlike all the others, this checks that the actual items are a
+        # superset of the expected items, rather than that they match.
+        for key, value in expected.items():
+            if value is not UNDEFINED:
```

Comment:
This `UNDEFINED` is a special sentinel for indicating that an item should not be present. There's not really any other way to do this in the code.

There's no way to get `ANY` to do the right thing here, since there's no way to distinguish between a `None` that comes from the caller, and a `None` that comes from a missed `get`.